lightning/src/chain/channelmonitor.rs

```diff
@@ -1810,9 +1816,9 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitor<Signer> {
 	/// `ChannelMonitor`. This is used to determine if an HTLC was removed from the channel prior
 	/// to the `ChannelManager` having been persisted.
 	///
-	/// This is similar to [`Self::get_pending_outbound_htlcs`] except it includes HTLCs which were
-	/// resolved by this `ChannelMonitor`.
-	pub(crate) fn get_all_current_outbound_htlcs(&self) -> HashMap<HTLCSource, HTLCOutputInCommitment> {
+	/// This is similar to [`Self::get_pending_or_resolved_outbound_htlcs`] except it includes
+	/// HTLCs which were resolved by this `ChannelMonitor`.
```

Comment:
Confused by this comment -- `get_pending_or_resolved_outbound_htlcs` now includes (at least some) resolved HTLCs, I think this needs updating?